packages/ui/src/theme/overwrites/components/Checkbox.ts

```diff
@@ -0,0 +1,42 @@
+function baseStyleControl(props: Record<string, unknown>) {
+  const { colorScheme: c } = props;
+  const disabledCbColorLight = 'gray.100';
+
+  return {
+    _checked: {
+      _active: {
+        bg: `${c}.700`,
```

Comment:
💭 It might be enough to write `brand.700` directly. For now, we don't have plans to customize checkbox colors.